README.md

```diff
@@ -191,12 +191,18 @@ OPTIONS
 
   -t, --templates=templates                [default: templates] Path to the templates folder
 
+  --devfile-registry=devfile-registry      [default: https://che-devfile-registry.openshift.io/] The URL of the Devfile
```

Comment:
To make it more explicit I would name the parameter `deploy-devfile-registry-url`

Reply:
Why `deploy-`? I agree `devfile-registry-url` would be more explicit and would explain the purpose more cleanly, but the `deploy-` prefix muddies the meaning for me again :)

I like @sleshchenko's idea for having `--deploy-devfile-registry=(true|false)` to deploy the devfile registry locally but IMHO that is out of scope of this issue.

But given the above, `--deploy-devfile-registry-url` is just not clear to me - am I deploying something using it or am I setting some configuration property? or both?

Reply:
I've renamed the arguments to `--devfile-registry-url` and `--plugin-registry-url`. 

I'm looking forward to having `--deploy-*-registry` but I took the liberty of not doing that in the scope of this PR.